templates/commands/render/action.go

```diff
@@ -137,7 +141,12 @@ func templateAndCompileRegexes(regexes []model.String, scope *common.Scope) ([]*
 
 // processGlobs processes a list of input String paths for simple file globbing.
 // Used after processPaths where applicable.
-func processGlobs(ctx context.Context, paths []model.String, fromDir string) ([]model.String, error) {
+func processGlobs(ctx context.Context, paths []model.String, fromDir string, skipGlobs bool) ([]model.String, error) {
+	// This supports older api_versions which didn't have glob support.
+	if skipGlobs {
+		return paths, nil
```

Comment:
I'm having second thoughts about this now. IIUC, this function will return relative paths of skipGlobs==true, otherwise absolute paths. Would it be possible to make it always return absolute paths? That would be less confusing for future maintainers. That would also avoid the need for the filepath.Join at line 64.

If you take this route, let's also document the absolute vs relative return values in the doc comment for this function.

Reply:
Changed to convert rel -> abs